[BUGFIX] Options not set for typo3_console and coreapi detection

jacobsenj opened the pull request:
This fixes the missing options for typo3_console and coreapi extension
detection for CompareDatabaseTask and FlushCachesTask (#8).

SomeBdyElse commented:
This looks similar to my own pull request #6

jacobsenj commented:
Indeed, but you missed one file ;)

helhum commented:
@jacobsenj @SomeBdyElse This indeed looks like a nobrainer fix. Can you both acknowledge on a canonical fix and push a rebased version of it (and then close the stale pull request).

I will then merge it asap.

Thanks

SomeBdyElse commented:
@helhum Please use this pull request. It comprises my own pull request entirely.

jacobsenj committed:
[BUGFIX] Options not set for typo3_console and coreapi detection

This fixes the missing options for typo3_console and coreapi extension
detection for CompareDatabaseTask and FlushCachesTask.

jacobsenj commented:
I rebased the two commits into one.

helhum commented:
Thanks!
